src/any_agent/callbacks/span_generation/smolagents.py

```diff
@@ -46,8 +46,8 @@ def after_llm_call(self, context: Context, *args, **kwargs) -> Context:
         output_tokens = 0
         if raw := response.raw:
             if token_usage := raw.get("usage", None):
-                input_tokens = token_usage.prompt_tokens
-                output_tokens = token_usage.completion_tokens
+                input_tokens = token_usage.get("prompt_tokens", None)
+                output_tokens = token_usage.get("completion_tokens", None)
```

Comment:
Did the type of `token_usage` changed in smolagents? I didn't see any CI failures